pkg/curatedpackages/packageclient.go

```diff
@@ -116,7 +117,56 @@ func (pc *PackageClient) InstallPackage(ctx context.Context, bp *packagesv1.Bund
 	if err != nil {
 		return err
 	}
-	fmt.Println(&stdOut)
+	fmt.Print(&stdOut)
+	return nil
+}
+
+func (pc *PackageClient) ApplyPackages(ctx context.Context, fileName string, kubeConfig string) error {
+	params := []string{"apply", "-f", fileName, "--kubeconfig", kubeConfig}
+	stdOut, err := pc.kubectl.ExecuteCommand(ctx, params...)
+	if err != nil {
+		fmt.Print(&stdOut)
+		return err
+	}
+	fmt.Print(&stdOut)
+	return nil
+}
+
+func (pc *PackageClient) CreatePackages(ctx context.Context, fileName string, kubeConfig string) error {
+	params := []string{"create", "-f", fileName, "--kubeconfig", kubeConfig}
+	stdOut, err := pc.kubectl.ExecuteCommand(ctx, params...)
+	if err != nil {
+		fmt.Print(&stdOut)
+		return err
+	}
+	fmt.Print(&stdOut)
+	return nil
+}
+
+func (pc *PackageClient) DeletePackages(ctx context.Context, packages []string, kubeConfig string) error {
+	params := []string{"delete", "packages", "--kubeconfig", kubeConfig, "--namespace", constants.EksaPackagesName}
+	params = append(params, packages...)
+	stdOut, err := pc.kubectl.ExecuteCommand(ctx, params...)
+	if err != nil {
+		fmt.Print(&stdOut)
+		return err
+	}
+	fmt.Print(&stdOut)
+	return nil
+}
```

Comment:
```suggestion
func (pc *PackageClient) DeletePackages(ctx context.Context, kubeConfig string, packageNames string...) error {
	params := []string{"delete", "packages", "--kubeconfig", kubeConfig, "--namespace", constants.EksaPackagesName}
	params = append(params, packageNames...)
	stdOut, err := pc.kubectl.ExecuteCommand(ctx, params...)
	if err != nil {
		fmt.Print(&stdOut)
		return err
	}
	fmt.Print(&stdOut)
	return nil
}
```
These change are to match the changes suggested in a previous commit.

But using a splat at the end, you buy yourself extra flexibility in how you call the methods.